pkg/virtctl/scp/scp.go

```diff
@@ -116,3 +131,29 @@ func usage() string {
   # Copy a file from the remote location to a local folder
   {{ProgramName}} scp jdoe@testvmi:myfile.bin ~/myfile.bin`
 }
+
+func ParseArguments(arg1 string, arg2 string) (local LocalArgument, remote RemoteArgument, toRemote bool, err error) {
+	remoteArg := arg1
+	localArg := arg2
```

Comment:
Rename arg1 and arg2 respectively in the func signature?

Reply:
how about renaming arg1 to `source` and arg2 renaming to `destination`
  -  arg1 → source (represents the source argument for the operation).
  -   arg2 → destination (represents the destination argument).